src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/SyntaxToken.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: BOM check

Reply:
This BOM looks correct in VS Code (UTF8 with BOM). Maybe it was wrong before my change?

Reply:
So weird. It's definitely UTF-8 (BOM) in main, but if I pull your branch, it is too? *shrug*